tests/rptest/services/redpanda.py

```diff
@@ -4094,20 +4161,42 @@ def is_allowed_log_line(line: str) -> bool:
                 raise NodeCrash(crashes)
 
     def raw_metrics(
-        self, node, metrics_endpoint: MetricsEndpoint = MetricsEndpoint.METRICS
+        self,
+        node: ClusterNode,
+        metrics_endpoint: MetricsEndpoint = MetricsEndpoint.METRICS,
+        query_string: str = "",
     ):
         assert node in self._started, f"Node {node.account.hostname} is not started"
 
-        url = f"http://{node.account.hostname}:9644/{metrics_endpoint.value}"
-        resp = requests.get(url, timeout=10)
+        url = f"http://{node.account.hostname}:9644/{metrics_endpoint.value}{query_string}"
```

Comment:
Not the most canonical way to use query strings with `requests` but I guess easier like this for compatibility with the k8s curl hacks.